server/routes/overview/educationAndTrainingView.ts

```diff
@@ -1,10 +1,17 @@
-import type { FunctionalSkills, InPrisonEducationRecords, EducationAndTraining, PrisonerSummary } from 'viewModels'
+import type {
+  FunctionalSkills,
+  InPrisonEducationRecords,
+  EducationAndTraining,
+  PrisonerSummary,
+  InPrisonEducation,
+} from 'viewModels'
 
 export default class EducationAndTrainingView {
   constructor(
     private readonly prisonerSummary: PrisonerSummary,
     private readonly functionalSkills: FunctionalSkills,
     private readonly completedInPrisonEducation: InPrisonEducationRecords,
+    private readonly completedEducationRecordsWithinLast12Months: InPrisonEducation[],
```

Comment:
Why is this new property an array, but the previous one (the one that we will be replacing) is an instance of the wrapper `InPrisonEducationRecords`
I think you're new property needs to be an instance of the wrapper, so that it includes the `problemRetrievingData` property